### ISSUE TITLE
This Apple DA: Probe SSC on //c for connected The Cricket!

### ISSUE THREAD

**Issue by frankmilliron** [a2stuff/a2d#760]
Probe all SSCs on a //c or //c+ for a connected The Cricket! clock and report just after SSC entry in 'This Apple' DA.

**Comment by inexorabletash** [comment]
Sample code:

https://github.com/a2stuff/prodos-drivers/blob/5211dc629611d8e55742347c5c8e5530e613e0a4/clocks/cricket/cricket.system.s#L55

There could be a generalization of the slot 3 special cases. "If slot N had device X, try these extra things..."

**Comment by frankmilliron** [comment]
Yeah I noticed that and `test.s` when I poked around the time-setting utilities.

**Comment by inexorabletash** [comment]
...and more generally, it'd be good to detect other clocks, e.g. NSC.
